tests/e2e_logging.py

```diff
@@ -1260,6 +1298,7 @@ def run(args):
             network = test_liveness(network, args)
         if args.package == "liblogging":
             network = test_receipts(network, args)
+            network = test_all_receipts(network, args)
```

Comment:
There's a chance that neither of these actually fetches a _signature_ receipt (if the `test_all` runs at the end of a large test where signatures are sparse, and we get unlucky with our sampling). Can we extend one of them to explicitly fetch a signature receipt (for the commit index, whatever it is), and even confirm that it has the structure we expect (empty proof and `leaf == root`)?

Reply:
I had the same thought, but for some reason decided it was hard to deliberately get a receipt for a signature tx, probably because it was late! You're absolutely right, always including the last commit does guarantee it will be a signature.